test/libsolidity/ABIJson/errors.sol

```diff
@@ -0,0 +1,88 @@
+error FreeError();
+contract X {
+    error E(uint);
+    error E2(uint a, uint b);
+    error E4(uint a, bytes[][] c);
+    struct S { uint x; }
+    error E5(S t);
```

Comment:
Can you add some other examples that have different `type` and `internalType`? E.g. an enum parameter?

Reply:
Not sure this is useful here, because it is just the same function.